### DISCUSSION TITLE
z into subdirectory of pwd?

### DISCUSSION THREAD

**Discussion by tennox** [ajeetdsouza/zoxide#903] [Q&A]
I don't find a way to z/zi into a subdirectory of pwd (by match, somewhere nested, not a direct child) - I've tried:
- `z . foo`
- `z foo .`

**Comment by tennox** [comment]
Best I found is literally
`z $PWD foo` :frowning_face:

**Comment by rieje** [comment]
What's wrong with ` z ./foo`?

**Reply by tennox** [reply]
I'm sorry I did not clarify my wish / use-case:

I meant to find by match, somewhere nested, not a direct child.
